src/Service/user-service.js

```diff
@@ -0,0 +1,65 @@
+const userModel = require('../Models/user-model');
+const bcrypt = require('bcryptjs')
+const tokenService = require('./token-service');
+const UserDto = require('../Modules/Dto/user-dto');
+
+class UserService {
+    async registration(login, password) {
+        const customer = await userModel.findOne({login});
+        if(customer) {
+            throw ApiError.BadRequest('Пользователь с таким логином уже существует!');
+        }
+        const hashPassword = await bcrypt.hash(password, 3);
+        const user = await userModel.create({login, password: hashPassword});
+        const userDto = new userDto(user);
+        const tokens = tokenService.generateTokens({...userDto});
+        await tokenService.saveToken(userDto.id, tokens.refreshToken);
+        return {
+            ...tokens,
+            user: userDto,
+        };
+    }
+
+    async actionLogin(login, password) {
```

Comment:
why actionLogin? In controllers, you access this service as "login" and here actionLogin. And does it work?